src/tools/miri/src/shims/foreign_items.rs

```diff
@@ -136,15 +138,29 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 // Find it if it was not cached.
                 let mut instance_and_crate: Option<(ty::Instance<'_>, CrateNum)> = None;
                 helpers::iter_exported_symbols(tcx, |cnum, def_id| {
+                    if tcx.is_foreign_item(def_id) {
+                        // Skip over imports of items
+                        return interp_ok(());
+                    }
+
                     let attrs = tcx.codegen_fn_attrs(def_id);
+                    // FIXME use tcx.symbol_name(instance) instead
```

Comment:
I'm not sure how to understand this comment. Is there some reason this can't just be changed now?

Reply:
`iter_exported_symbols` returns `DefId`'s, not `Instance`'s like `tcx.symbol_name()` needs.